algosdk/abi/method.py

```diff
@@ -0,0 +1,219 @@
+import json
```

Comment:
Similar remarks regarding type hints and `@classmethod` vs. `@staticmethod` as previous.